src/main/scripts/src/app/workspace/organization/organization-settings.component.ts

```diff
@@ -17,10 +17,106 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component} from '@angular/core';
+import {Component, OnInit, Output, EventEmitter, ViewChild, ElementRef} from '@angular/core';
+import {ActivatedRoute, ParamMap, Router} from '@angular/router';
+
+import {Organization} from '../../core/dto/organization';
+import {WorkspaceService} from '../../core/workspace.service';
+import {OrganizationService} from '../../core/rest/organization.service';
+import {HttpResponse} from '@angular/common/http';
+import {ProjectService} from '../../core/rest/project.service';
+import {Project} from "../../core/dto/project";
 
 @Component({
-  templateUrl: './organization-settings.component.html'
+  templateUrl: './organization-settings.component.html',
+  styleUrls: ['./organization-settings.component.scss']
 })
-export class OrganizationSettingsComponent {
+export class OrganizationSettingsComponent implements OnInit {
+
+  private organization: Organization;
+  private organizationCode: string;
+  private errorMessage: any;
+  private originalOrganizationCode: string;
+  private projectsCount: number;
+  private organizationDescriptionEditable: boolean = false;
+
+  @ViewChild('organizationDescription')
+  public organizationDescription: ElementRef;
+
+  constructor(private organizationService: OrganizationService,
+              private workspaceService: WorkspaceService,
+              private route: ActivatedRoute,
+              private router: Router,
+              private projectService: ProjectService) {
+  }
+
+  public ngOnInit(): void {
+    this.organization = new Organization();
```

Comment:
It's a good idea to initialize it with some values, so that when data gets loaded, there is no flashing happening as a result of elements appearing.

```
this.organization = {
  color: '#cccccc' // grey,
  icon: 'fa fa-exclamation-circle', // exclamation mark
    .
    .
    .
}
```

And to pair with it a method that should be used instead of checks for `undefined` or `null` value of `organization`:
```
public initialized(): boolean {
  return this.organization.color !== '#cccccc' && 
             this.organization.icon !== 'fa fa-exclamation-circle' &&
                        .
                        .
                        .
}
```